src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,252 @@
+            const params = new URLSearchParams({
+                'ownerAddress': 'erd1yj4jlay9rrzahgsg9frxw6l5qyca9dqhp8c4f5e0vdsytwkvl',
+            });
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toEqual("Validation failed for argument 'ownerAddress' (a bech32 address is expected)");
+                });
+        });
+
+        [
+            {
+                filter: 'sort',
+                value: 'balance',
+            },
+        ].forEach(({ filter, value }) => {
+            describe(`when filter ${filter} is applied`, () => {
+                it(`should return accounts details based on ${filter} with value ${value} and ordered descendent `, async () => {
+                    const order = 'desc';
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${filter}=${value}&${order}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toBeDefined();
+                            expect(Number(res.body[0].balance)).toBeGreaterThan(Number(res.body[1].balance));
+                        });
+                });
+            });
+        });
+
+        [
+            {
+                filter: 'sort',
+                value: 'timestamp',
+            },
+        ].forEach(({ filter, value }) => {
+            describe(`when filter ${filter} is applied`, () => {
+                it(`should return accounts details based on ${filter} with value ${value} and ordered descendent `, async () => {
+                    const order = 'desc';
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${filter}=${value}&${order}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toBeDefined();
+                            expect(res.body[0].timestamp).toBeGreaterThanOrEqual(res.body[1].timestamp);
+                        });
+                });
+            });
+        });
+
+        test.each`
+        size
+        ${1}
+        ${55}
+        ${4}
+        ${10000}`
+            (
+                `should return a list of $size items`,
+                async ({ size }) => {
+                    const params = new URLSearchParams({
+                        'size': size,
+                    });
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${params}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toHaveLength(size);
+                        });
+                }
+            );
+    });
+
+    describe('/accounts/count', () => {
+        it('should return count of all blocks from all shards', async () => {
```

Comment:
This is not the correct test case description. You need to add description for accounts